src/app/shared/components/workshop-checkbox-dropdown/workshop-checkbox-dropdown.component.scss

```diff
@@ -19,6 +19,10 @@
   padding: 1rem;
 }
 
+.label-disabled {
+  opacity: 0.3;
```

Comment:
could you please check if we use such styles for all disabled fields to know if we can move it out and reuse